pkg/deploy/reconcile_manager.go

```diff
@@ -21,9 +21,9 @@ import (
 
 type Reconcilable interface {
 	// Reconcile object.
-	Reconcile(ctx *DeployContext) (result reconcile.Result, done bool, err error)
+	Reconcile(ctx *DeployContext) (reconcile.Result, bool, error)
```

Comment:
Why did you remove variables names? They show purpose of each arg and even if 2 of 3 are obvious, I'd keep it for the `done` one.